openff/toolkit/typing/engines/smirnoff/parameters.py

```diff
@@ -3625,9 +3625,12 @@ def create_force(self, system, topology, **kwargs):
                 #                             "must be provided")
             else:
                 force.setNonbondedMethod(openmm.NonbondedForce.LJPME)
-                force.setCutoffDistance(9.0 * unit.angstrom)
+                force.setCutoffDistance(self.cutoff)
                 force.setEwaldErrorTolerance(1.0e-4)
 
+                force.setSwitchingDistance(self.switch_width)
+                force.setUseSwitchingFunction(self.switch_width < self.cutoff)
```

Comment:
It seems to me that having a larger switch_width than the cutoff should raise an error -- Silently disabling the switching it would likely be the more surprising behavior.